release-calendar/src/client/components/Calendar.tsx

```diff
@@ -68,12 +70,18 @@ export class Calendar extends React.Component<CalendarProps, CalendarState> {
           this.props.singleRelease,
         );
         this.setState({singleEvents: getSingleReleaseEvents(release)});
+        this.gotoDate(release[0].start);
       } else {
         this.setState({singleEvents: []});
       }
     }
   }
 
+  gotoDate = (date: Date): void => {
```

Comment:
Is there a reason why you use an anonymous function here and not a regular function syntax? (`gotoDate(date: Date): void {`)

Anonymous functions are more useful when you use them as, say, an argument to another function (e.g., `arr.sort((firstEl, secondEl) => { ... })`)

Reply:
No there wasn't any reason. I just wasn't aware one was the "regular", and I've gotten used to the anonymous way since I use functions as arguments more often. I'll switch this one to the regular function syntax!